symedocs.yml

```diff
@@ -3,250 +3,34 @@ channels:
-  - pillow=8.1.0=py39hf95b381_2
-  - pip=21.0.1=pyhd8ed1ab_0
-  - prometheus_client=0.9.0=pyhd3deb0d_0
-  - prompt-toolkit=3.0.16=pyha770c72_0
-  - prompt_toolkit=3.0.16=hd8ed1ab_0
-  - pthread-stubs=0.4=h36c2ea0_1001
-  - ptyprocess=0.7.0=pyhd3deb0d_0
-  - pugixml=1.11.4=h9c3ff4c_0
-  - pybtex=0.24.0=py39hf3d152e_0
-  - pybtex-docutils=1.0.0=py39hf3d152e_0
-  - pycparser=2.20=pyh9f0ad1d_2
-  - pygments=2.7.4=pyhd8ed1ab_0
-  - pyopenssl=20.0.1=pyhd8ed1ab_0
-  - pyparsing=2.4.7=pyh9f0ad1d_0
-  - pyqt=5.12.3=py39hf3d152e_7
-  - pyqt-impl=5.12.3=py39h0fcd23e_7
-  - pyqt5-sip=4.19.18=py39he80948d_7
-  - pyqtchart=5.12=py39h0fcd23e_7
-  - pyqtwebengine=5.12.1=py39h0fcd23e_7
-  - pyrsistent=0.17.3=py39h3811e60_2
-  - pysocks=1.7.1=py39hf3d152e_3
-  - python=3.9.1=hffdb5ce_5_cpython
-  - python-dateutil=2.8.1=py_0
-  - python-symengine=0.6.1=py39h68068d8_3
-  - python_abi=3.9=1_cp39
-  - pytz=2021.1=pyhd8ed1ab_0
-  - pyyaml=5.4.1=py39h3811e60_0
-  - pyzmq=22.0.1=py39hea8fd45_0
-  - qt=5.12.9=h9d6b050_2
-  - qtconsole=5.0.2=pyhd8ed1ab_0
-  - qtpy=1.9.0=py_0
-  - rb-bundler=2.1.4=hd4c1694_0
-  - readline=8.0=he28a2e2_2
-  - requests=2.25.1=pyhd3deb0d_0
-  - ruby=2.5.7=he592edb_2
-  - send2trash=1.5.0=py_0
-  - setuptools=49.6.0=py39hf3d152e_3
-  - shellingham=1.4.0=pyh44b312d_0
-  - six=1.15.0=pyh9f0ad1d_0
-  - smmap=3.0.5=pyh44b312d_0
-  - sniffio=1.2.0=py39hf3d152e_1
-  - snowballstemmer=2.1.0=pyhd8ed1ab_0
-  - sphinx=3.4.3=pyhd8ed1ab_0
-  - sphinx-autobuild=2020.9.1=pyhd8ed1ab_0
-  - sphinx-copybutton=0.3.1=pyhd8ed1ab_0
-  - sphinx-sitemap=2.2.0=pyhd8ed1ab_0
-  - sphinx-thebe=0.0.7=pyh9f0ad1d_0
-  - sphinx-togglebutton=0.2.2=pyh9f0ad1d_0
-  - sphinxcontrib-applehelp=1.0.2=py_0
-  - sphinxcontrib-bibtex=1.0.0=py_0
-  - sphinxcontrib-devhelp=1.0.2=py_0
-  - sphinxcontrib-htmlhelp=1.0.3=py_0
-  - sphinxcontrib-jsmath=1.0.1=py_0
-  - sphinxcontrib-qthelp=1.0.3=py_0
-  - sphinxcontrib-serializinghtml=1.1.4=py_0
-  - sqlalchemy=1.3.23=py39h3811e60_0
-  - sqlite=3.34.0=h74cdb3f_0
-  - symengine=0.6.0=h800644d_1
-  - sysroot_linux-64=2.12=h77966d4_13
-  - terminado=0.9.2=py39hf3d152e_0
-  - testpath=0.4.4=py_0
-  - tk=8.6.10=h21135ba_1
-  - toml=0.10.2=pyhd8ed1ab_0
-  - tornado=6.1=py39h3811e60_1
-  - traitlets=5.0.5=py_0
-  - tzdata=2021a=he74cb21_0
-  - urllib3=1.26.3=pyhd8ed1ab_0
-  - wcwidth=0.2.5=pyh9f0ad1d_2
-  - webencodings=0.5.1=py_1
-  - wheel=0.36.2=pyhd3deb0d_0
-  - widgetsnbextension=3.5.1=py39hf3d152e_4
-  - xeus=1.0.0=h78d96c3_0
-  - xeus-cling=0.12.0=h5a79028_0
-  - xorg-libxau=1.0.9=h7f98852_0
-  - xorg-libxdmcp=1.1.3=h7f98852_0
-  - xtensor=0.23.1=h4bd325d_0
-  - xtl=0.7.2=h4bd325d_0
-  - xz=5.2.5=h516909a_1
-  - yaml=0.2.5=h516909a_0
-  - zeromq=4.3.4=h9c3ff4c_0
-  - zipp=3.4.0=py_0
-  - zlib=1.2.11=h516909a_1010
-  - zstd=1.4.8=ha95c52a_1
+  - doxyrest
+  - ipykernel
+  - ipywidgets
+  - jupyter
+  - jupyter_nbextensions_configurator
+  - jupyterlab
+  - jupytext
+  - matplotlib
+  - myst-nb
+  - numpy
+  - pandas
```

Comment:
pandas?

Reply:
Was planning to use it in a tutorial for `symengine.py`. I'll remove it for now.